topsort-endpoints.yml

```diff
@@ -208,6 +208,12 @@ components:
           type: string
           format: string
           example: v_89kjm
+        quality:
+          type: number
```

Comment:
Please add a description of what this field means (the same for products above). As it is right now is impossible to know what one should put into that field. Is 1 the best quality or the lowest?